x-pack/filebeat/processors/decode_cef/cef/cef.rl

```diff
@@ -97,7 +97,7 @@ func (e *Event) unpack(data string) error {
             fhold; fnext gobble_extension;
         }
         action recover_next_extension {
-            extKey, extValueStart, extValueEnd = "", 0, 0
+            extKey, extValueStart, extValueEnd, escapes = "", 0, 0, escapes[:0]
```

Comment:
It would be nice to have these non-ragel state variables have some commentary in the decl above.

Reply:
I did a little bit of refactoring to make things slightly more readable. Will you look again please?